cluster-operator/src/main/java/io/strimzi/operator/cluster/Main.java

```diff
@@ -47,8 +47,17 @@
 @SuppressFBWarnings("DM_EXIT")
 public class Main {
     private static final Logger LOGGER = LogManager.getLogger(Main.class.getName());
+    private static Vertx vertx = Vertx.vertx(
+        new VertxOptions().setMetricsOptions(
+            new MicrometerMetricsOptions()
+                .setPrometheusOptions(new VertxPrometheusOptions().setEnabled(true))
+                .setJvmMetricsEnabled(true)
+                .setEnabled(true)));
```

Comment:
IIRC, this did not worked well in general to maker this static. It was causing all kind of problems during tests.

Reply:
OK, will remove static.